src/Crowdsale/DynamicPriceSale.sol

```diff
@@ -70,8 +70,10 @@ contract DynamicSale is ContextUpgradeable, ReentrancyGuardUpgradeable, Ownable2
         quote = IERC20Upgradeable(quote_);
         minter = IMinterDAO(minter_);
         currentPrice = 222 ether;
-        saleHardCap = 7000 ether;
```

Comment:
Why this change? Does the whitepaper specify this saleHardCap?

Reply:
Let me review this, I think I changed to test something did not change it back